airflow/providers/slack/operators/slack.py

```diff
@@ -89,9 +92,9 @@ def execute(self, **kwargs):  # noqa: D403
 
 class SlackAPIPostOperator(SlackAPIOperator):
     """
-    Posts messages to a slack channel
-    Examples:
+    Posts messages to a slack channel.
 
+    Examples:
     .. code-block:: python
```

Comment:
Tests is failing because of this, a space is needed here

```suggestion
    Examples:

    .. code-block:: python
```

Reply:
@kaxil 
I merged this suggestion and run pre-commit.

I found that this suggestion is illegal about D412(No blank lines allowed between a section header and its content)

Reply:
So I think that 
if I obey D412, I can't pass the static check,
if i pass the static check, I can't obey D412 (not confirmed.)

Reply:
Don't worry, I will check back later tonight or tomorrow and we will figure out a solution, you have done an amazing job with this PR, thanks.


Btw there are some conflicts, can you rebase on the lastest master please

Reply:
I see. Thank you.

I've rebased and pushed.